src/methods/clientMethods.mjs

```diff
@@ -137,17 +139,55 @@ export async function getOptionsForFilters(params) {
 }
 
 export async function getSeance(params) {
-  const { seanceId } = params;
+  const { seanceId, userId } = params;
   const seance = await Seance.findById(seanceId);
   const movieTheater = await MovieTheater.findOne(
     { 'halls._id': seance.hallId },
     { halls: { $elemMatch: { _id: seance.hallId } } }
   ).select('-seances');
-  console.log('seance', seance);
-  console.log('movieTheater', movieTheater);
-
+  const nowTime = new Date();
+  nowTime.toISOString();
```

Comment:
const nowTime = new Date();
  nowTime.toISOString();

Reply:
absolutely useless

Reply:
Agree with you! Removed it